build/documentation/generate-typedocs.js

```diff
@@ -40,11 +44,19 @@ process.argv.forEach(function (val, index) {
     var validArg = true;
 
     if (index === 2) {
-        val === "--dry-run" ? dryRun = true : validArg = false;
```

Comment:
You might consider using https://github.com/yargs/yargs - we've already got it as a dependency and it makes working with process arugments a little bit easier.